code/modules/antagonists/heretic/magic/mind_gate.dm

```diff
@@ -0,0 +1,52 @@
+/datum/action/cooldown/spell/pointed/mind_gate
+	name = "Mind Gate"
+	desc = "Opens your mind, and theirs."
```

Comment:
Spell description should say what it's actually doing

Reply:
Fixed